### PR TITLE
Express Get JSON Nathan Chon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
image

### CODE REVIEW

File: express-get-json/server.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+const app = express();
+
+const array = [];
+
+app.use((req, res, next) => {
+
+  array.push(grades[12]);
+  array.push(grades[47]);
+  array.push(grades[273]);
```
Comment (TimelikeClosure) [inline express-get-json/server.js:32]: @Nathan-Chon Please make sure you're creating a new array each time a request is made.

File: express-get-json/server.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  res.json(array);
+  next();
```
Comment (TimelikeClosure) [inline express-get-json/server.js:35]: In general, it's not a good idea to call `next()` unless there are additional middleware that need to run.

File: express-get-json/server.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const app = express();
+
+const array = [];
+
+app.use((req, res, next) => {
```
Comment (TimelikeClosure) [inline express-get-json/server.js:28]: Please make sure you're using the correct `app` method and parameters to only respond to `GET /api/grades` requests.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by Nathan-Chon** [comment]
image
Revised all changes necessary

**Pull request merged**